docs/faq.rst

```diff
@@ -702,6 +702,29 @@ Now Borg will be bandwidth limited. Nice thing about pv is that you can change r
 .. _pipeviewer: http://www.ivarch.com/programs/pv.shtml
 
 
+How can I restore from a BTRFS snapshot to the original path?
+-------------------------------------------------------------
+
+When I do a backup of a BTRFS file system I first do a readonly snapshot and
+then back it up:
```

Comment:
You could replace this with a list of use cases:
- btrfs snapshots
- mounting a filesystem to another place than usual

Reply:
OK